Add placeholder download video button

adamjarling opened the pull request:
# Summary 
This PR wires up the download video button for a Filetset on the Structure tab. This is not fully complete until we get the download url (coming soon in another issue)

<img width="1176" alt="image" src="https://github.com/nulib/meadow/assets/3020266/74de2dca-cba3-4255-b4f9-7b9d6eb7be8a">


# Specific Changes in this PR
- Add download button to the Work > Structure tab for `video` mime type files

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.

- [x] Patch
- [ ] Minor
- [ ] Major

# Steps to Test
1. Go to a Video Work page, click on the "Structure" tab and notice "Download video" button for an "Access" Fileset video.
2. Go to an Audio Work page and notice the "Download video" button doesn't appear in the same spot as above.

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
  - [ ] Database Schema changes
  - [ ] GraphQL API
  - [ ] Elasticsearch API
  - [ ] Ingest Sheet
  - [ ] CSV metadata export/update API
  - [ ] Shared Links export API
- Backwards-incompatible API changes
  - [ ] Database Schema changes
  - [ ] GraphQL API
  - [ ] Elasticsearch API
  - [ ] Ingest Sheet
  - [ ] CSV metadata export/update API
  - [ ] Shared Links export API
- [ ] Requires data migration
- [ ] Requires database triggers disabled during deployment/migration
- [ ] Requires reindex
- [ ] Terraform changes
  - [ ] Adds/requires new or changed Terraform variables
- [ ] Pipeline configuration changes (requires `mix meadow.pipeline.setup` run)
- [ ] Requires new variable added to `miscellany`
- [ ] Specific deployment synchronization instructions with other apps/API's
- [ ] Other specific instructions/tasks


# Tested/Verified
- [ ] End users/stakeholders

kdid commented on app/assets/js/components/Work/Fileset/ActionButtons/Access.jsx:
```diff
@@ -29,11 +46,19 @@ function MediaButtons({ fileSet }) {
+          <span className="icon">
+            <IconDownload />
+          </span>
+          Download Video
```
@adamjarling -This could also be an audio file rather than video...

(The backend ticket spelled it out better that it was functionality for AV access files. https://github.com/nulib/repodev_planning_and_docs/issues/3292 )

adamjarling replied:
Ah, gotcha.  Easy enough change.  Thanks.

adamjarling commented:
@kdid This is ready to review with your latest suggestion.  Thanks again. 👍

adamjarling commented:
> 

@kdid Yea, that was bugging me as well, but it's because it has an icon in the button, and the icon class (from Bulma) has a height on it).  I could always remove the icon, or yea you're right I'll figure it out before merging 😀

kdid commented:
@adamjarling - Yeah, the edit one could get a pencil icon I guess. For some reason the buttons higher up on the page "Publish/Unpublish" and "Get Shared Link" line up even though one has an icon and one doesn't.

adamjarling commented:
> @adamjarling - Yeah, the edit one could get a pencil icon I guess. For some reason the buttons higher up on the page "Publish/Unpublish" and "Get Shared Link" look like they line up even though one has an icon and one doesn't. Or maybe it's just that one is purple...

@kdid Button sizing now fixed on latest push, squashed commits. 👍
